### ISSUE TITLE
How can I clear the AJAX cache?

### ISSUE THREAD

**Issue by booruguru** [bootsz/wp-advanced-search#180]
WPAS appears to cache AJAX results, which means, changes to my loop aren't appearing properly. How can I clear/erase the cache?

**Comment by bootsz** [comment]
Yeah, I think that feature needs some revision as it's a bit too aggressive.  I think you can disable the caching by commenting out line 125 in js/scripts.js.  I'll need to think of a better approach going forward.
